frontend/src/components/Home.js

```diff
@@ -88,6 +101,10 @@ export default class Home extends Component {
                <div class="horizontalDisplay">
               <h3 class="h2Align">Recently Played</h3>
                 <div class= "recentPlayed">
```

Comment:
i believe that using this.props.image_url and this.props.title is the correct way to retrieve the spotify data from the backend but Kelly is going to look into it as well